test/servico/cst01Test.java

```diff
@@ -52,21 +53,26 @@ public void tearDown() {
     }
     
     @Test
-    public void testeNotificaçãoDeAusência_ÚnicoHorário() throws ParseException
+    public void testeNotificaçãoDeAusência_ÚnicoHorário() 

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/calebepb/substituicao/file/2765413210/issues/source?bid=3308597&fileBranchId=3353664#l56)